semantic_conventions/trace/rpc.yaml

```
@@ -124,3 +125,26 @@ groups:
         note: >
           This is always required for jsonrpc. See the note in the general
           RPC conventions for more information.
+  - id: rpc.grpc.message
+    prefix: "message" # TODO: Change the prefix to rpc.grpc.message?
```

```suggestion
    prefix: "message"  # TODO: Change the prefix to rpc.grpc.message?
```
To make yamllint happy.